fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -135,6 +135,10 @@ public class FedoraPropsConfig extends BasePropsConfig {
     @Value("${fcrepo.cache.webac.acl.timeout.minutes:10}")
     private long webacCacheTimeout;
 
+    @Value("${fcrepo.register.banner:on}")
+    private String fcrepoRegisterBanner;
```

Comment:
Testing instructions suggested the property was `fcrepo.banner.display` but it is actually `fcrepo.register.banner`. I think going closer to the first and using the `enabled` that we do for other things (drawing in the idea of using a boolean). Something like `fcrepo.banner.enabled=true|false`

Reply:
Good points. I'll also remove the superflous 'fcrepo' from the java attribute names and getter so that it becomes ```bannerEnabled```  to follow the other property to attribute mappings within the class.

Reply:
No I would not remove the `fcrepo` as if someone assigns this on the java command line it could be unclear what it is for.
```bash
> ... -Dbanner.enabled=false
```
versus
```bash
> ... -Dfcrepo.banner.enabled=false
```

Reply:
I am refering to the java attributes, not the java properties. So:
```fcrepoRegisterBanner```
vs
```registerEnable``` 
within the java source ```fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java``` 
Agreed that the fcrepo prefix should stay for the property

Hope that makes sense

Reply:
Ahh okay